### DISCUSSION TITLE
Full output into /dev/null

### DISCUSSION THREAD

**Discussion by martinscaune-mintos** [mydumper/mydumper#925] [Q&A] [answered]
Hi, 

Is there a way to direct all output data genereted by mydumper to `/dev/null` ? My use case is to warm up the DB by performing a full dump, so I don't really need any of the data to be saved on the instance, but if I just specify `-o /dev/null` it results in error as mydumper can't create any files there. And when using `--steam` there are still a copy written on the instance that fills up the space quickly. 

Thanks.

**Comment by davidducos** [comment]
Hi @martinscaune-mintos, 
I never heard about your use case! For sure, MyDumper was not designed to do something like that. However, I think that you will be able to achieve what you want. You should check https://www.percona.com/blog/upload-ongoing-mydumper-backups-to-s3/ which mentions --exec-per-threads. I think that you will be able to create a bash script that could redirect the STDIN to /dev/null, so at the end mydumper is going to create empty files. 
So, the parameter that you to add to mydumper will be --exec-per-threads="/usr/bin/bash script.sh"
and the script.sh might be something like:
```
while read nothing
do
echo -n ""
done
```

**Comment by davidducos** [comment, marked as answer]
If you combine --stream and -F, the file size on the temporary disk path will be limited to -F multiply the amount of threads. 
I think that this should work for your use case.
